build.sbt

```diff
@@ -1,3 +1,5 @@
+name := "metals"
```

Comment:
when a project is opened in intellij now it gets gibberish name, with this change it will be displayed in a list of projects as 'metals' and it doesn't change anything in regard to functionality